lib/perl/Genome/InstrumentData/AlignmentResult/Merged/BamQc.pm

```diff
@@ -160,9 +148,12 @@ sub create {
     );
 
     # use per lane instrument data id as link name
-    if ($lane_flag) {
-        my $instr_data = $self->alignment_result->instrument_data;
+    my @instrument_data = $self->alignment_result->instrument_data;
+    my $lane_flag;
+    if (@instrument_data == 1) {
+        my $instr_data = $instrument_data[0];
         $bam_qc_params{bam_link_name} = $instr_data->id;
+        $lane_flag = 1;
```

Comment:
Is it okay that the `$lane_flag` is now being set on merged results with a single instrument data whereas it was not before?